src/components/StatusButton/__test__/index.test.jsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import StatusButton from "..";
+
+const MockStatusButton = () => {
+  return <StatusButton id="3456" />;
+};
+
+describe("Status button component", () => {
+  it("should render status button correctly", () => {
+    render(<StatusButton />);
+    const statusButton = screen.getByTestId("active-status");
+    expect(statusButton).toBeInTheDocument();
+  });
+
+  it("on hover show status list", () => {
```

Comment:
~~hover~~ click

Reply:
when a user hovers over the dropdown icon it show the list